src/sidebar/helpers/groups.ts

```diff
@@ -18,6 +18,8 @@ function allowLeavingGroups(settings: SidebarSettings): boolean {
   return !!config.allowLeavingGroups;
 }
 
+export const WORLD_GROUP_ID = '__world__';
```

Comment:
`PUBLIC_GROUP_ID` might be a better name, since that is what the group is called for users. The ID of course we have to keep as-is.